server/src/test/kotlin/org/javacs/kt/LintTest.kt

```diff
@@ -26,4 +30,27 @@ class LintTest : SingleFileTestFixture("lint", "LintErrors.kt") {
         assertThat(diagnostics, not(empty<Diagnostic>()))
         assertThat(languageServer.textDocumentService.lintCount, lessThan(5))
     }
+
+    @Test fun `lintting should not be dropped if another lintting is running`() {
+        var callbackCount = 0;
+        languageServer.textDocumentService.debounceLint.waitForPendingTask()
+        languageServer.textDocumentService.setTestLintRecompilationCallback({
```

Comment:
I couldn't figure out a better way to do the test that to add a hook (and the test still seems somewhat fragile). Ideas on better testing would be welcome!